src/variety/slither.js

```diff
@@ -93,6 +118,11 @@
 		}
 	},
 
+	"Cell@swslither": {
+		maxnum: 6,
+		minnum: 0
```

Comment:
`minnum` is the same as regular Cell, so it doesn't need to be copied

Reply:
Done